File: pset1/cash/cash.c
#include <cs50.h>
#include <stdio.h>
#include <math.h>

int main(void)
{
    int coins = 0;
    float dollars = 0;
    do
    {
    dollars  = get_float("Change,$: ");   
    }
    while (dollars < 0);
    int cents = round(dollars * 100);
    while (cents > 0)
    {
        if (cents >= 25)
        {
            coins += 1;
            cents -= 25;
        }
        else if (cents >= 10)
        {
            coins += 1;
            cents -= 10;
        }
        else if (cents >= 5)
        {
            coins += 1;
            cents -= 5;
        }
        else if (cents >= 1)
        {
            coins += 1;
            cents -= 1;
        }
             
    }
    printf("I have, %i coin(s)\n", coins);
}

File: pset6/cash.py
from cs50 import get_float

while True:

    dollars = get_float("Change owed,$: ")

    if dollars > 0:
        break

cents = dollars * 100
coins = 0

while cents > 0:
    if cents >= 25:
        coins += 1
        cents -= 25

    elif cents >= 10:
        coins += 1
        cents -= 10

    elif cents >= 5:
        coins += 1
        cents -= 5

    elif cents >= 1:
        coins += 1
        cents -= 1
print(coins);